### ISSUE TITLE
AsyReader fails on lines with patterns/styles

### ISSUE THREAD

**Issue by hb020** [nunobrum/spicelib#64]
when an asy has a line like 

```LINE Normal -16 89 -16 96 2```

spicelib\spicelib\editor\asy_reader.py:71

```python
                elif line.startswith("LINE"):
                    tag, weight, x1, y1, x2, y2 = line.split()
```
```python
Exception has occurred: ValueError       (note: full exception trace is shown but execution is paused at: _run_module_as_main)
too many values to unpack (expected 6)
  File "...spicelib/spicelib/editor/asy_reader.py", line 72, in __init__
    tag, weight, x1, y1, x2, y2 = line.split()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
....
```

asc_editor handles this correctly. The same is likely to happen with rectangle, circle and arc.

Will provide a PR for this.

**Comment by hb020** [comment]
PR (#65) created

**Comment by nunobrum** [comment]
Weird bug. I wonder why this was not caught before. I did test this before committing.

**Comment by hb020** [comment]
A fair part of the parser in asceditor is also applicable to asyeditor. It was just not ported. So I did that, and added warnings in the code. (and adapted the qsch writer)

**Comment by nunobrum** [comment]
Merged #65
